frontend/views/containers/chatroom/MessageNotification.vue

```diff
@@ -61,18 +63,28 @@ export default ({
           [MESSAGE_NOTIFICATIONS.UPDATE_DESCRIPTION]:
             L('Updated the channel description to: {description}', { description: channelDescription }),
           [MESSAGE_NOTIFICATIONS.DELETE_CHANNEL]: L('Deleted the channel: {title}', { title: channelName }),
-          [MESSAGE_NOTIFICATIONS.VOTE]: L('Voted on “{title}”', { title: '' }) // TODO: polls are not implemented yet
+          [MESSAGE_NOTIFICATIONS.VOTE_ON_POLL]: L('Voted on {options}', { options: votedOptions }),
+          [MESSAGE_NOTIFICATIONS.CHANGE_VOTE_ON_POLL]: L('Changed votes to {options}', { options: votedOptions })
         }
       }
 
       const notificationSelector = this.isGroupDirectMessage() ? 'onGroupDM' : 'default'
       const text = notificationTemplates[notificationSelector][this.notification.type]
       return { text }
+    },
+    isPollNotification () {
+      return [
+        MESSAGE_NOTIFICATIONS.VOTE_ON_POLL,
+        MESSAGE_NOTIFICATIONS.CHANGE_VOTE_ON_POLL
+      ].includes(this.notification.type)
     }
   },
   methods: {
     addEmoticon (emoticon) {
       this.$emit('add-emoticon', emoticon)
+    },
+    jumpToPoll () {
+      alert('TODO: implement "jumping to a particular message."')
```

Comment:
Do you think you can open an issue for this?

Reply:
I opened an issue: https://github.com/okTurtles/group-income/issues/1674

Can you add a comment here linking to that issue?

Reply:
@taoeffect thnx